server/src/main/java/org/cloudfoundry/identity/uaa/util/PasswordValidatorUtil.java

```diff
@@ -46,13 +46,23 @@ public static PropertiesMessageResolver messageResolver(String messagesResourceP
         }
     }
 
+    public static PasswordValidator secretValidator(GenericPasswordPolicy<?> policy,
```

Comment:
created 2 methods to have it separated that for users a hard coded minimum password needs 1 char 
For clients this is not true we had and can have empty secrets.

The min length is validated with the client.secret.policy .... settings, but here the default for minLength is 0 , so should be alloed